### PR TITLE
Applying updated designs to docs repo

### PR DESCRIPTION
**Pull request by adamcbrewer**
Merging existing code into the docs repo and applying the recent design updates.

Rebased commits:

+ Nav fix
+ Homepage styles
+ Who page
+ Navigation active state
+ Fixing FAQ page layout
+ Fixing indentations
+ Fixes for resources page
+ Refactor and removing roadmap from sidebar nav

### CODE REVIEW

**Comment by adamcbrewer** [comment]
Hey @pbakaus – could you make sure to verify my username (working on behalf of Toaster) for this PR? Thanks!

Just so you're aware, I'm away from the end of today and won't have computer access for the next couple of weeks, but @elijahzapien will be helping out, so please let him know if there are any issues. I've added him as a collaborator to my repo, so he should be be able to push changes from my fork.

There might be some discrepancies with the recent amends and the designs that were approved, so It might be worthwhile contacting Rozalia about these. Elijah will be able to sort out anything that's flagged!

File: views/base.html [hunk 1]
```diff
@@ -33,7 +37,9 @@
           <li><a href="{{ g.doc(header_doc.fields.who.get('href'), locale=doc.locale).url.path }}">{{_(header_doc.fields.who.get('copy'))}}</a></li>
           <li><a href="{{header_doc.fields.blog.get('href')}}">{{_(header_doc.fields.blog.get('copy'))}}</a></li>
           <li><a href="{{header_doc.fields.github.get('href')}}">{{_(header_doc.fields.github.get('copy'))}}</a></li>
+          {# Is this being removed?
```
Comment (pbakaus) [inline views/base.html]: Why did you remove the link to the roadmap?

**Comment by pbakaus** [comment]
Thanks a lot @adamcbrewer, and thanks for taking over, @elijahzapien! Changes mostly look fine, except the following:

- Why is the link to the roadmap gone? (left a comment inline as well) How do we navigate to it instead?
- Not sure if this was intended, but the roadmap layout looks slightly broken – the bubbly list is supposed to be in the middle, like here: https://www.ampproject.org/roadmap/. If intended, did you discuss the design change with any team members?

Everything else looks good to me!

**Comment by pbakaus** [comment]
Ping!

**Comment by pbakaus** [comment]
I've manually merged this myself as I didn't want to wait any longer – I reenabled the roadmap link, and applied some of the new roadmap styles, but not all. Happy to discuss with make sense, as soon as you find the time. Thanks!
